recipe/deploy/update_code.php

```diff
@@ -108,6 +108,7 @@
         cd('{{release_path}}');
         run("$git clone -l $bare .");
         run("$git remote set-url origin $repository", ['env' => $env]);
+        $target = get('branch', 'HEAD');
```

Comment:
Target variable already defined and can contain tags and commits ref.

Reply:
hi @antonmedv do you have a suggestion to change this?
when @jhhazelaar was working on the original pull request you gave this comment: https://github.com/deployphp/deployer/pull/3466/files/e427f9a6e6805ea110add81cdb221df8d5440fea#r1089138042

Reply:
We cant change it

Reply:
Hi @antonmedv 
I understand, can we perhaps implement it in a seperate if statement?

Reply:
We can try

Reply:
Hi @antonmedv 
I have added a new option called `force_head` to allow the users to set a boolean. 
Hope you like it!

Reply:
Will review it.